travis/test-spark-autologging.sh

```diff
@@ -6,7 +6,7 @@ set -ex
 
 # Build Java package
 pushd mlflow/java/spark
-mvn package -DskipTests
+mvn package -DskipTests -q
```

Comment:
Build package silently (only print errors) to reduce build log size

Reply:
Awesome!